resources/views/customer/receipt/proforma/mail.blade.php

```diff
@@ -0,0 +1,19 @@
+<style>
+    th, td {
+       text-align: center;
+    }
+</style>
+@component('mail::message')
+Good day,
+
+This is your credit receipt.
+
+@component('mail::table')
+| Due Date | Amount | Tax | Remark |
+| ------------- |:-------------:| ------------- | ------------- |
+|{{$proforma['due_date']}} | {{$proforma['amount']}} | {{$proforma['tax']}} | {{$proforma['remark']}} |
```

Comment:
Similar as I explained in receipts.
![image](https://user-images.githubusercontent.com/32955000/184778859-29d3e283-4891-427c-a003-415aeb960b4c.png)